roles/libvirt_manager/tasks/create_cloud_init_iso.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Generate cloud-init iso
+  become: false
+  vars:
+    _images_dir: "{{ [ cifmw_libvirt_manager_basedir, 'workload', ] | ansible.builtin.path_join }}"
+  block:
+    - name: "Acquire authorized_keys from hypervisor"
+      ansible.builtin.slurp:
+        src: "{{ ansible_user_dir ~ '/.ssh/authorized_keys' }}"
+      register: _authorized_keys_b64
+
+    - name: Create a list of ssh pub keys from authorized_keys
+      ansible.builtin.set_fact:
+        _ssh_authorized_keys: |-
+          [
+          {% for ssh_key in (_authorized_keys_b64.content | b64decode).split('\n') | select() %}
+            "{{ ssh_key | trim }}",
+          {% endfor %}
+          ]
+
+    - name: "Generate instance random UUID"
+      ansible.builtin.set_fact:
+        _instance_uuid: "{{ 99999999 | random(seed=vm) | to_uuid | lower }}"
```

Comment:
Do we really need a fact for this?

Reply:
I created it to have the vars while calling the role as concise as possible. 

Yesterday I learned that defining a var using `set_fact` with its default, is just _yet another way_ to define a variable (at host level but still). A fact becomes an `ansible_fact` (with its pro and cons) only defining `cacheable` property as `true`. Interesting, isn't it? https://docs.ansible.com/ansible/latest/collections/ansible/builtin/set_fact_module.html#parameter-cacheable

Reply:
ack